build/loc.proj

```diff
@@ -73,6 +72,15 @@
     <Copy SourceFiles="@(EulaFiles->'%(Identity)')" DestinationFiles="@(EulaFiles->'%(DestinationDir)')"/>
   </Target>
 
+  <Target Name="MoveLcgFilesToArtifacts" AfterTargets="Localize" Condition="'$(BuildRTM)' != 'true'">
+    <ItemGroup>
+      <_MoveLcgFiles Include="@(LocalizedToolFiles)" Condition="'%(LocalizedToolFiles.Extension)'== '.lcg'">
+        <DestinationDir>$(ArtifactsDirectory)$([MSBuild]::MakeRelative($(ArtifactsDirectory)localize\%(LocalizedToolFiles.lang), %(LocalizedToolFiles.RootDir)%(LocalizedToolFiles.Directory)))%(LocalizedToolFiles.Culture)\%(LocalizedToolFiles.Filename)%(LocalizedToolFiles.Extension)</DestinationDir>
```

Comment:
Moving file from "E:\A\_work\30\s\artifacts\localize\ENU\NuGet.Console\15.0\bin\release\NuGet.Console.dll.lcg" to "E:\A\_work\30\s\artifacts\NuGet.Console\15.0\bin\release\en\NuGet.Console.dll.lcg".
2017-05-10T22:00:58.8848823Z   Moving file from "E:\A\_work\30\s\artifacts\localize\ENU\NuGet.Tools\15.0\bin\release\NuGet.Tools.dll.lcg" to "E:\A\_work\30\s\artifacts\NuGet.Tools\15.0\bin\release\en\NuGet.Tools.dll.lcg".